docs/topics/maven.md

```diff
@@ -262,15 +262,15 @@ The following attributes are supported:
 
 ### Attributes common to JVM and JS
 
-| Name | Property name | Description | Possible values |Default value |
-|------|---------------|-------------|-----------------|--------------|
-| `nowarn` | | Generate no warnings | true, false | false |
-| `languageVersion` | kotlin.compiler.languageVersion | Provide source compatibility with the specified version of Kotlin | "1.4" (DEPRECATED), "1.5", "1.6", "1.7", "1.8", "1.9" | 
-| `apiVersion` | kotlin.compiler.apiVersion | Allow using declarations only from the specified version of bundled libraries | "1.3" (DEPRECATED), "1.4" (DEPRECATED), "1.5", "1.6", "1.7", "1.8", "1.9" | 
-| `sourceDirs` | | The directories containing the source files to compile | | The project source roots
-| `compilerPlugins` | | Enabled compiler plugins  | | []
-| `pluginOptions` | | Options for compiler plugins  | | []
-| `args` | | Additional compiler arguments | | []
+| Name | Property name | Description | Possible values                                                                          |Default value |
```

Comment:
The same note as above:

<img width="1548" alt="Screenshot 2023-02-09 at 15 03 59" src="https://user-images.githubusercontent.com/78360457/217836419-dbd4d5e3-5a95-47af-b1f2-a842a23860ea.png">